File: 3.2P/Implementation/ClockImplementation/Counter.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ClockImplementation
{
    public class Counter
    {
        private int _count = 0;
        public int Count
        {
            get { return _count; }
            set { _count = value; }
        }

        public void Increment()
        {
            _count += 1;
        }

        public void Reset()
        {
            _count = 0;
        }
    }
    
}

File: 10.1P/#--Java/ClockJava/Counter.java
public class Counter{
    private int _count = 0;

    public void setCount(int value)
    {
        _count = value;
    }

    public int getCount()
    {
        return _count;
    }

    public Counter()
    {

    }
    public void Tick()
    {
        _count += 1;
    }
    public void Reset()
    {
        _count = 0;
    }
}